docker-compose.yml

```diff
@@ -33,3 +33,74 @@ services:
       - 6832:6832/udp
       - 5778:5778
       - 16686:16686
+  etcd0:
+    image: quay.io/coreos/etcd
+    ports:
+      - "2379:2379"
+    volumes:
+      - etcd0:/etcd_data
```

Comment:
Do we need mounted volume for etcd? I think it can slow down mac/windows user a bit.

Reply:
@t-tomalak I don't think so (for now). Because we're using etcd not as a storage, but just during a runtime operation of stashing a module